### PR TITLE
docs: update `install` section of README.md

### PR DESCRIPTION
**Pull request by KrishPatel13**
**What kind of change does this PR introduce?**
Bugfix: Update install section in README.md

**Summary**

- [x] Add TesseractOCR Setup

**Checklist**
* [x] My code follows the style guidelines of OpenAdapt
* [x] I have performed a self-review of my code
* [x] If applicable, I have added tests to prove my fix is functional/effective
* [x] I have linted my code locally prior to submission
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation (e.g. README.md, requirements.txt)
* [x] New and existing unit tests pass locally with my changes

**How can your code be run and tested?**
Follow https://github.com/OpenAdaptAI/OpenAdapt#install on a machine without tesseract.

### CODE REVIEW

**Comment by Mustaballer** [comment]
@KrishPatel13 This wouldn't be a `fix`, but should be replaced with `docs`, since this PR is Documentation-only changes. Refer to CONTRIBUTING.md for more detail

File: README.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ The direction is adjacent to [Adept.ai](https://adept.ai/), with some key differ
 
 ## Install
 
+Required dependencies:
+- Python 3.10
+- Git
+- Tesseract (for OCR)
+
+Follow the setup instructions in setup.md for your operating system.
```
Comment (abrichr) [inline README.md:42]: @KrishPatel13 is there a setup.md?

Comment (KrishPatel13) [reply README.md:42]: I will be adding soon.

Comment (KrishPatel13) [reply README.md:42]: Addressed in: [8dc42c1](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/8dc42c1feefd4e39aab70f1ff453a15872832dfb)

**Comment by KrishPatel13** [comment]
@abrichr Ready for review!

**Comment by KrishPatel13** [comment]
>> https://github.com/OpenAdaptAI/OpenAdapt/pull/324#issuecomment-1619317249

First point addressed in: [f4957b2](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/f4957b27569e23b8affde5e8131809d817ce3e75)

Second point:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,15 +28,15 @@ from openadapt import models
 In order to effectively communicate any bugs or request new features, please select the appropriate form. If none of the options suit your needs, you can click on "Open a blank issue" located at the bottom.
 
 ## Testing
-[GitHub Actions](https://github.com/MLDSAI/OpenAdapt/actions/new) are automatically run on each pull request to ensure consistent behaviour and style. The Actions are composed of PyTest, [black](https://github.com/psf/black) and [flake8](https://flake8.pycqa.org/en/latest/user/index.html).
+[GitHub Actions](https://github.com/MLDSAI/OpenAdapt/actions/new) are automatically run on each pull request to ensure consistent behavior and style. The Actions are composed of PyTest, [black](https://github.com/psf/black) and [flake8](https://flake8.pycqa.org/en/latest/user/index.html).
 
-You can run these tests on your own computer by downloading the depencencies in requirements.txt and then running pytest in the root directory. 
+You can run these tests on your own computer by downloading the dependencies in requirements.txt and then running `pytest` in the root directory.
```
Comment (Mustaballer) [inline CONTRIBUTING.md:33]: If we are getting rid of requirements.txt, you'll have to update this

Comment (KrishPatel13) [reply CONTRIBUTING.md:33]: Great Point @Mustaballer,

@abrichr 
Should I remove requirements.txt in my PR which is updating the README ?
And then update README as well as remove the .venv setup ?

**Comment by KrishPatel13** [comment]
@abrichr Ready for review!

File: CONTRIBUTING.md (hunk 2 above)
Comment (KrishPatel13) [reply CONTRIBUTING.md]: Addressed in: [7e07cf4](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/7e07cf4b82c1f891d302727129b5d70018dc049c)

File: README.md [hunk 3]
```diff
@@ -32,10 +32,50 @@ The direction is adjacent to [Adept.ai](https://adept.ai/), with some key differ
 3. OpenAdapt works with all types of desktop GUIs, including virtualized (e.g. Citrix) and web
 4. OpenAdapt is open source! (license TBD, please see https://github.com/MLDSAI/OpenAdapt/issues/246)
 
+
 ## Install
 
+<br/>
+
+| Installation Method  |               [Installation Scripts](https://openadapt.ai/)                |                               Manual Setup                                |
```
Comment (abrichr) [inline README.md:40]: Can you please link to https://openadapt.ai/#start ?

File: README.md [hunk 4]
```diff
@@ -32,10 +32,50 @@ The direction is adjacent to [Adept.ai](https://adept.ai/), with some key differ
...
+  Start-Process powershell -Verb RunAs -ArgumentList '-NoExit', '-ExecutionPolicy', 'Bypass', '-Command', "iwr -UseBasicParsing -Uri 'https://raw.githubusercontent.com/MLDSAI/OpenAdapt/main/install/install_openadapt.ps1' | Invoke-Expression"
+  </pre>
+
+#### MacOS
+- Download and install Git and Python 3.10
+- Press Command+Space, type "terminal", and press Enter
+- Copy and paste the following command into the terminal, and press Enter:
+  <pre className="whitespace-pre-wrap code text-slate-600 bg-slate-100 p-3 m-2">
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/OpenAdaptAI/install/HEAD/install_puterbot.sh)"
+  </pre>
```
Comment (abrichr) [inline README.md:62]: @KrishPatel13 can you please update these instructions to match https://github.com/OpenAdaptAI/OpenAdaptWeb/pull/30/files ?

Comment (KrishPatel13) [reply README.md:62]: Addressed in: [f8d37ff](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/f8d37ffd3651d164d682203380fa246aea5bc57a)

The website is not uptodate:

image

Comment (KrishPatel13) [reply README.md:62]: Updated in favour of: https://github.com/OpenAdaptAI/OpenAdaptWeb/pull/30/files#r1252126510 and https://github.com/OpenAdaptAI/OpenAdaptWeb/pull/30/files#r1252128457

File: .gitignore [hunk 5]
```diff
@@ -5,7 +5,6 @@
 __pycache__
 cache
 *.egg-info
-.venv
```
Comment (KrishPatel13) [inline .gitignore:8]: remove .venv in favour of: https://github.com/OpenAdaptAI/OpenAdapt/pull/289/files#r1252004135

File: .gitignore [hunk 6]
```diff
@@ -19,7 +18,7 @@ cache
 .VSCode
 .vsCode
 
-#Idea
+# Idea
```
Comment (KrishPatel13) [inline .gitignore:21]: add a space to maintain the format

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -28,15 +28,15 @@ from openadapt import models
 In order to effectively communicate any bugs or request new features, please select the appropriate form. If none of the options suit your needs, you can click on "Open a blank issue" located at the bottom.
 
 ## Testing
-[GitHub Actions](https://github.com/MLDSAI/OpenAdapt/actions/new) are automatically run on each pull request to ensure consistent behaviour and style. The Actions are composed of PyTest, [black](https://github.com/psf/black) and [flake8](https://flake8.pycqa.org/en/latest/user/index.html).
+[GitHub Actions](https://github.com/MLDSAI/OpenAdapt/actions/new) are automatically run on each pull request to ensure consistent behavior and style. The Actions are composed of PyTest, [black](https://github.com/psf/black) and [flake8](https://flake8.pycqa.org/en/latest/user/index.html).
```
Comment (KrishPatel13) [inline CONTRIBUTING.md:31]: some typos fixed

**Review by KrishPatel13 — COMMENTED** [review]
Self-reviewed!

**Comment by KrishPatel13** [comment]
Ready for merging! @abrichr

File: .gitignore (hunk 5 above)
Comment (abrichr) [reply .gitignore:8]: See https://github.com/OpenAdaptAI/OpenAdapt/pull/289/files#r1252150636

File: README.md [hunk 8]
```diff
@@ -32,10 +32,50 @@ The direction is adjacent to [Adept.ai](https://adept.ai/), with some key differ
...
+
+### Manual Setup
+
+Prerequisite:
+- Python 3.10
+- Git
+- Tesseract (for OCR)
 
-Install with [Poetry](https://python-poetry.org/) (recommended):
+For the setup of any/all of the above dependencies, follow the steps [here](./SETUP.md).
```
Comment (abrichr) [inline README.md:74]: How about `in [SETUP.md](./SETUP.md)`?

File: SETUP.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+
+
+
+## Tesseract OCR
+
+#### Windows:
+- Download the following binary (.exe) file:
+    [Tesseract Setup](https://digi.bib.uni-mannheim.de/tesseract/tesseract-ocr-w64-setup-5.3.1.20230401.exe)
+- Run the setup file and Install the Tesseract OCR
```
Comment (abrichr) [inline SETUP.md:50]: How about:

```
- Run the downloaded file to install Tesseract OCR
```

File: SETUP.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+- Add Tesseract to User and System PATH variable.
+    Follow the following article for the above step.
```
Comment (abrichr) [inline SETUP.md:52]: How about we remove this line?

File: README.md (hunk 8 above)
Comment (KrishPatel13) [reply README.md]: [d9e3762](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/d9e37620d49368e385870217647d6a542f7d9059)

File: SETUP.md (hunk 9 above)
Comment (KrishPatel13) [reply SETUP.md]: Addressed in: [fbf264c](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/fbf264c22a4ff68818b3ef22f91049cccd96794c)

File: SETUP.md (hunk 10 above)
Comment (KrishPatel13) [reply SETUP.md]: Addressed in:

**Comment by KrishPatel13** [comment]
@abrichr addressed all the comments!

File: .gitignore (hunk 5 above)
Comment (KrishPatel13) [reply .gitignore:8]: RE-addressed in: [b103e18](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/b103e185667d42c426411f4f79f8c426d16f4d33)

Add `.venv` back.

**Pull request merged**